### ISSUE TITLE
Support for container style queries

### ISSUE THREAD

**Issue by 5t3ph** [parcel-bundler/lightningcss#467]
When trying to write a [container style query](https://developer.chrome.com/blog/style-queries/), the following error is received:

```js
Unexpected token Function("style")
```

Example rule:

```css
@container style(--dark-mode: true) {
  p {
    color: white;
  }
}
```

Style queries started to receive browser support as of Chromium 111 when used with custom properties, so I would expect to be able to write them.

I would appreciate at least a transform example as a temporary solution, but I was a bit confused trying to write that myself :)

Thanks, I've been enjoying using Lightning CSS!

**Comment by devongovett** [comment]
Recently implemented this in 7413eacf4c8756c5da82943edbfb82d43b52ee48 (#415). Will be released very soon.

**Comment by 5t3ph** [comment]
Great news, thanks @devongovett!
